microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1482,6 +1482,33 @@ Spatial input (mouse, touch, pen) is received by the application and must be sen
 
 ---
 
+
+<!-- ------------------------------ -->
+#### Drag and drop
+
+Dragging from WebView2 to another application is supported by default. However, dropping into WebView2 (Which includes drag/drop operations within WebView2 itself) requires forwarding IDropTarget events the host application receives from the system to WebView2 using the APIs below.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2CompositionController.DragLeave Method](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave?view=webview2-dotnet-1.0.1370.28&preserve-view=true#microsoft-web-webview2-core-corewebview2compositioncontroller-dragleave)
```

Comment:
```suggestion
* [CoreWebView2CompositionController.DragLeave Method](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave)
```

Reply:
test:
prefix + /dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave
=
learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave
Success: You end up at
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave?view=webview2-dotnet-1.0.1587.40